draft-ietf-quic-recovery.md

```diff
@@ -236,7 +237,9 @@ underestimation of min RTT, which in turn prevents underestimating smoothed RTT.
 Ack-based loss detection implements the spirit of TCP's Fast Retransmit
 {{?RFC5681}}, Early Retransmit {{?RFC5827}}, FACK, and SACK loss recovery
 {{?RFC6675}}. This section provides an overview of how these algorithms are
-implemented in QUIC.
+implemented in QUIC.  Though both time-based loss detection and early retransmit
+use a timer, they are part of Ack-based detection because they do not use a
```

Comment:
```suggestion
use a timer, they are part of ack-based detection because they do not use a
```

Reply:
the word is "acknowledgment", which I appreciate is long and hard to say and type, but we should use the whole word unless we're talking about the frame.

Reply:
Hmm, the naming of this is a good point, but I think that's best saved for a follow up change, since it's used in a good number of spots.  Also, I was thinking it's ACK like the frame, but I'm not sure we want to go to ACK-based loss detection everywhere?